### ISSUE TITLE
chai doesn't know BigInts are numbers

### ISSUE THREAD

**Issue by karlb** [chaijs/chai#1652]
I would expect these asserts to succeed:
```
> assert.isNumber(1n);
AssertionError: expected 1n to be a number
```

```
> assert.isAtMost(1n, 2n);
AssertionError: expected 1 to be a number or a date
```

**Comment by 43081j** [comment]
I think @koddsson did a pr related to this a while back

If I remember correctly, you would use `isNumeric` rather than `isNumber`

But all the value comparisons should probably just understand both

**Comment by koddsson** [comment]
I just published [`chai@5.1.2`](https://github.com/chaijs/chai/releases/tag/v5.1.2) which includes my recent work on `BigInt`. @43081j is right that we didn't add support to `isNumber` as that would be a breaking change but you can use `isNumeric` in the mean time. With that one change your example should work correctly 🕺🏻

**Comment by karlb** [comment]
Works great, thanks for fixing!
